README.md

```diff
@@ -1,2 +1,16 @@
 # templyte
-Junior Design Project
+Junior Design Project created by Team 7331
+
+The Templyte is a web application that allows users to upload, download, and share coding templates for starter coding projects. A coding template is a set of files that contains starter code for a programming project and has variables that the user can provide values for when the template is downloaded. 
+
+To launch templyte:
+Install Node and NPM
+	To install on mac: https://blog.teamtreehouse.com/install-node-js-npm-mac
+	To install on windows: https://blog.teamtreehouse.com/install-node-js-npm-windows
+Via command line:
+	npm install
+	npm start 
+	npm run development-server
+	npm run development-gui
```

Comment:
This is only for starting it in development mode - this should be specified and there should be a section for deployment